partials/content.html

```diff
@@ -5,3 +5,4 @@
 {% if asyncapi.hasChannels() %}
 {% include "./operations.html" %}
 {% endif %}
+{% include "./messages.html" %}
```

Comment:
It would be good to have `.hasMessages()` check here. But it doesn't exist now.
We can add that in the future.

Reply:
Please create an issue in parser to add it so we don't forget